ionoscloud/utils.go

```diff
@@ -46,17 +46,21 @@ func resourceIpFailoverImporter(ctx context.Context, d *schema.ResourceData, met
 		return nil, err
 	}
 
-	if lan.Properties.IpFailover != nil {
-		err := d.Set("ip", *(*lan.Properties.IpFailover)[0].Ip)
-		if err != nil {
-			return nil, err
-		}
-	}
+	failoverSlice := lan.Properties.IpFailover
```

Comment:
this import should be moved to the resource file, don't now why it has been left to utils